### PR TITLE
Updating `configutil` and all usage of it.

### PR DESCRIPTION
**Pull request by dhermes**
## PR Summary

 - **Type:** Internal
 - **Intended Change Level:** minor

#### Reviewers:

Reviewers keep the following in mind while reviewing this PR, and provide an assessment of them in your approval comment:

- Does the "Intended Change Level" above match the actual behavior of this PR?
- Is this PR following patterns set forth in the package (if modifying a package), or the go-sdk design patterns if implementing a new package from scratch?
- Does this require a backport to LTS versions? If so ping, let the contributors group know.

### CODE REVIEW

File: configutil/int64_test.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package configutil
+
+import (
+	"context"
+	"testing"
+
+	"github.com/blend/go-sdk/assert"
+)
+
+func TestInt646464646464(t *testing.T) {
```
Comment (wcharczuk) [inline configutil/int64_test.go:10]: hey is this the name from the monorepo?

Comment (wcharczuk) [reply configutil/int64_test.go:10]: i have confirmed, it is in fact, the name from the monorepo

Comment (wcharczuk) [inline configutil/int64_test.go:10]:
```suggestion
func TestInt64(t *testing.T) {
```

File: configutil/env.go [hunk 2]
```diff
@@ -133,3 +100,11 @@ func (e EnvVars) Duration() (*time.Duration, error) {
 	}
 	return nil, nil
 }
+
+// vars resolves the env vars
+func (e EnvVars) vars(ctx context.Context) env.Vars {
+	if vars := env.GetVars(ctx); vars != nil {
```
Comment (wcharczuk) [inline configutil/env.go:106]: did this function make it with the rest of the changes? I can't find it

Comment (dhermes) [reply configutil/env.go:106]: I presume "this function" == `env.GetVars()` here? If yes, I pulled in `env` changes in https://github.com/blend/go-sdk/pull/215.

File: configutil/file.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+)
+
+// File reads the string contents of a file as a literal config value.
+type File string
+
+// String returns the string contents of a file.
+func (f File) String(ctx context.Context) (*string, error) {
+	contents, err := ioutil.ReadFile(string(f))
+	if err != nil {
+		return nil, nil
```
Comment (mat285) [inline configutil/file.go:16]: Not returning the error?

Comment (dhermes) [reply configutil/file.go:16]: This is a copy-pasted port of existing code in our internal version of this. Happy to patch bugs in the implementation but would strongly prefer to do in a different PR.

Comment (wcharczuk) [reply configutil/file.go:16]: yeah it should ideally differentiate between `ENOTFOUND` and an actual error reading from disk, but for now we just skip over the variable source if there is any issue vs. failing resolving the config

**Pull request merged**